clusters/app.ci/hive/README.md

```diff
@@ -43,4 +43,131 @@ hiveadmission-9f7df866b-zb98l       1/1     Running   0          2m16s
 ```
 
 ## Deploy a cluster via Hive
-TBU
+
+### SSH key
+Managed by `ci-secret-bootstrap`: `jenkins-ci-iam-ssh-key`.
+
+### Pull Secret
+
+Create a secret for pull-secret which will be used by `ClusterDeployment` later.
+
+```bash
+### reuse the existing pull secret, we use this pull-secret to generate the install-config above.
+$ oc --context build01 extract secret/pull-secret -n openshift-config
+$ oc create secret generic test01-pull-secret --from-file=.dockerconfigjson=./.dockerconfigjson --type=kubernetes.io/dockerconfigjson -n hive
+```
+
+### AWS
+Account: https://openshift-ci.signin.aws.amazon.com/console
+
+This account has been used to create ephemeral clusters via installer directly for CI jobs.
+
+#### DNS
+base domain: `hive.aws.ci.openshift.org`.
+
+* route53 console: Create a zone `hive.aws.ci.openshift.org`. We can see the list of name servers in the NS record after creation.
```

Comment:
Why do we have to manually create a zone for each cluster?

Reply:
Not for each cluster. This is the basic domain for all clusters on an aws account. If we have more than one zones, we could choose which one to use.

For example, this one is from the demo below for its api server.

```
server: https://api.test01.hive.aws.ci.openshift.org:6443
```

Reply:
Okay, can you clarify that and split this document into a "Configuring Hive" and "Setting up a new cluster" section?

Reply:
Updated.